README.md

```diff
@@ -175,6 +175,8 @@ export class CounterState {
 ```
 
 ## Payload type safety 
+`Emittable` and `EmitterAction` are generics which allow you to type the payload, which is `void` (alias not present) by default. \\
+Eg. `Emittable` is equal to `Emittable<void>`, you can type it like `Emittable<number>` or `Emittable<any>` (if you want to accept all possible values).
```

Comment:
For example, `Emittable` equals to the `Emittable<void>`. Also it's possible to type it like `Emittable<number>` or `Emittable<any>` (if you want it to accept all possible values).